pyproject.toml

```diff
@@ -17,10 +17,12 @@ good-names = ["df", "ca", "tp", "to", "x", "y", "i", "j", "f", "q", "ax", "pa",
 max-args = 10
 max-attributes = 15
 max-statements = 100
+extension-pkg-allow-list = "pydantic.dataclasses"
```

Comment:
It allow pylint to import the module and check what it contains because it can't do it with static analysis because it's a C extension, this fixes false negatives from pylint stating the dataclasses module doesn't have members